wbooks/android/app/src/main/java/com/wbooks/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.lugg.ReactNativeConfig.ReactNativeConfigPackage;
```

Comment:
No!

Reply:
![image](https://user-images.githubusercontent.com/45558924/106556155-b9b3bd80-64fd-11eb-907e-2c39b2c23104.png)

Listed in the library's repository as manual link for android

Reply:
We are not using the manual link, RN has the autolink feature for 0.60+ version